packages/babel-helpers/src/helpers/using.js

```diff
@@ -0,0 +1,19 @@
+/* @minVersion 7.0.0-beta.0 */
+
+export default function _using(stack, value, isAwait) {
+  if (value !== null && value !== void 0) {
```

Comment:
According to [7.5.4 AddDisposableResource](https://arai-a.github.io/ecma262-compare/?pr=3000&id=sec-adddisposableresource&secAll=true&secSubtree=true), we should throw a `TypeError` when `value` is not an object.